libs/feature/record/src/lib/record-meta/record-meta.component.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnChanges,
+  OnDestroy,
+} from '@angular/core'
+import { Meta } from '@angular/platform-browser'
+import {
+  DatasetRecord,
+  ServiceRecord,
+} from '@geonetwork-ui/common/domain/model/record'
+
+@Component({
+  selector: 'gn-ui-record-meta',
+  template: '<>',
```

Comment:
it would be great if you could make this component standalone, thanks!

Reply:
Set as standalone and removed <> as it was adding this to html page :p